video/cloud-client/faces/faces.py

```diff
@@ -66,10 +64,12 @@ def analyze_faces(path):
         print('Thumbnail size: {}'.format(len(face.thumbnail)))
 
         for segment_id, segment in enumerate(face.segments):
-            print('Track {}: {} to {}'.format(
-                segment_id,
-                segment.start_time_offset,
-                segment.end_time_offset))
+            start_time = (segment.segment.start_time_offset.seconds +
+                          segment.segment.end_time_offset.nanos / 1e9)
```

Comment:
Should this be "start_time_offset.nanos"?

Reply:
good catch, thanks!  i will scan through the other files and correct them.